src/main/java/io/quarkus/search/app/quarkusio/QuarkusIO.java

```diff
@@ -126,12 +164,16 @@ private Stream<Guide> versionedGuides() throws IOException {
                     Stream<Guide> quarkusGuides = parseYamlMetadata(webUri, quarkus, quarkusVersion)
                             .flatMap(guide -> translateGuide(guide, translations));
                     if (Files.exists(quarkiverse)) {
-                        // the full content won't be translated, but the title/summary may be, so we want to get that info out if available
-                        // we also have to create "copies" of quarkiverse guides for other languages, otherwise we won't find them in search results
-                        // as we are using a `must` filter on language:
-                        Map<Language, Catalog> quarkiverseTranslations = translations(
-                                (directory, language) -> resolveTranslationPath(p.getFileName().toString(),
-                                        quarkiverse.getFileName().toString(), directory, language));
+                        // TODO: to be removed, but NOTE, that it means that localized sites may
+                        //  have a translated title/description for a quarkiverse guide,
+                        //  but in search we'd be using english version only, as we are not "copy-pasting" quarkiverse guide entities ... ?
+
+                        //                        // the full content won't be translated, but the title/summary may be, so we want to get that info out if available
+                        //                        // we also have to create "copies" of quarkiverse guides for other languages, otherwise we won't find them in search results
+                        //                        // as we are using a `must` filter on language:
+                        //                        Map<Language, Catalog> quarkiverseTranslations = translations(
+                        //                                (directory, language) -> resolveTranslationPath(p.getFileName().toString(),
+                        //                                        quarkiverse.getFileName().toString(), directory, language));
```

Comment:
while looking into warnings, I've noticed this part being unused (translations of quarkiverse metadata elements).
hence I wonder.... do we keep it as it is, or should we return back the part about cloning quarkiverse guides for all languages 🙈 oooor make a "rule" that in localized sites quarkiverse guides should remain untranslated (even their titles and descriptions)?

Reply:
Okay, I didn't catch that titles/summaries of Quarkiverse guides were actually translated in some cases... are you sure they are? I mean, they could be in theory, but are there any translations in practice?

If so, I guess we either have to revert my work, or make the `Guide#title` and `Guide#summary` a `Map<Language, String>` to account for the fact that for the same URL we may have multiple titles/summaries. It *should* be possible to handle that in the current bridge. Hopefully.

But really this sounds like it belongs in a separate PR :)

Reply:
yeah 😃 

soooo looking at https://ja.quarkus.io/version/main/guides/ if I do a cntrl+f for `Quarkiverse Hub` there are mostly translated guides 🙈

Reply:
Alright, I'll have a look then

Reply:
Created #170